cdap-ui/app/cdap/components/AbstractWidget/KeyValueDropdownWidget/index.tsx

```diff
@@ -72,3 +74,12 @@ export default function KeyValueDropdownWidget(props) {
 }
 
 (KeyValueDropdownWidget as any).propTypes = WIDGET_PROPTYPES;
+(KeyValueDropdownWidget as any).getWidgetAttributes = () => {
+  return {
+    'key-placeholder': { type: 'string', required: false },
+    'value-placeholder': { type: 'string', required: false },
```

Comment:
I don't see where we look for/use this?

Reply:
Yes! We wouldn't need value-placeholder since value is a dropdown for `KeyValueDropdownWidget`.